### DISCUSSION TITLE
Minidexed fits into the original Pi case and only a trained eye could tell that it's modified!

### DISCUSSION THREAD

**Discussion by bearwolf** [probonopd/MiniDexed#277] [Show and tell]
![IMG_3594](https://user-images.githubusercontent.com/3290087/172449948-decfe767-80cf-4fd0-afe7-f1138e0806e3.jpg)
![IMG_3595](https://user-images.githubusercontent.com/3290087/172449954-666f5e0c-1d73-4c6d-bb9b-a06fe1ee5fb8.jpg)

Okay okay... Gonna look into how to sketch up a case for 3d printing... The one i like on thingieverse is for the 4 and not the 3 i have so it needs some modifications ...

**Comment by Banana71** [comment]
And the winner for the Scotch® 3M Tape🎶 Award goes to.....🥳

**Reply by probonopd** [reply]
Do they make shrink tubing in that size? ;-)

**Comment by Banana71** [comment]
i would also be very interested in a 3d printed case and pcb board

**Comment by sou7611** [comment]
Yes, but you'd need a flamethrower to get enough heat on it to shrink it...

**Comment by bedgin** [comment]
Laughed my ass off when I clicked into this with anticipation. lol Well played, Sir, Well played!

**Comment by Doug2233** [comment]
Strangely attractive ….

**Comment by mongio** [comment]
Well, I Just did it for real, My raspi came in kit with a sort of camera case, I used the camera hole for the encoder, a small 0,96 inch oled display (128*64) into the fan space and the dac glued somewhere inside :)
![image](https://user-images.githubusercontent.com/14731010/204639843-250b44bc-0e11-45d3-a2f0-bd874b427e46.jpg)
